src/academy/devdojo/maratonajava/javacore/streams/test/StreamCollectorsTest01.java

```diff
@@ -0,0 +1,48 @@
+package academy.devdojo.maratonajava.javacore.streams.test;
+
+import academy.devdojo.maratonajava.javacore.streams.domain.LightNovel;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.DoubleSummaryStatistics;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class StreamCollectorsTest01 {
+
+    private static List<LightNovel> lightNovels = new ArrayList<>(List.of(
+            new LightNovel("Death Note", 3.99),
+            new LightNovel("Dragon Ball", 2.59),
+            new LightNovel("Demon Slayer", 5.89),
+            new LightNovel("Demon Slayer", 5.89),
+            new LightNovel("Pokemon", 6.50),
+            new LightNovel("Fullmetal alchemist", 5.99),
+            new LightNovel("Monogatari", 4.99),
+            new LightNovel("Overlord", 8.99)
+    ));
+
+    public static void main(String[] args) {
+        System.out.println(lightNovels.stream().count()); //count
+        System.out.println(lightNovels.stream().collect(Collectors.counting())); //count using Collectors
+
+        System.out.println("------------");
+
+        lightNovels.stream().max(Comparator.comparing(LightNovel::getPrice)).ifPresent(System.out::println); //Max price
+        lightNovels.stream().collect(Collectors.maxBy(Comparator.comparing(LightNovel::getPrice))).ifPresent(System.out::println); //Max price with Collectors
```

Comment:
Can you scanner this Class with SonarLint?

Reply:
Yes, but in this case I need to follow the course structure